.travis.yml

```diff
@@ -3,10 +3,18 @@ python:
   - 3.4
 addons:
   apt:
+    sources:
+      - ubuntu-toolchain-r-test
```

Comment:
Is this a well supported PPA with active development? Are we pretty sure that it will remain reliable?